Models/TrackModels/TrackModelMapper.swift

```diff
@@ -18,8 +18,8 @@ final class TrackModelMapper: TrackModelMapperProtocol {
     
     func mapAPIToUI(apiTrack: APITrackModel) -> TrackModel {
         
-        let trackModelMapper = TrackModel(name: apiTrack.name)
-        return trackModelMapper
+        let track = TrackModel(name: apiTrack.name, duration: apiTrack.duration)
+        return track
```

Comment:
you can directly return TrackModel without saving it to a constant. 
`return TrackModel(name: apiTrack.name, duration: apiTrack.duration)`